projects/aws/image-builder/builder/utils.go

```diff
@@ -7,9 +7,9 @@ import (
 	"os/exec"
 )
 
-func cloneRepo(cloneUrl, destination, tag string) error {
+func cloneRepo(cloneUrl, destination string) error {
 	log.Print("Cloning eks-anywhere-build-tooling...")
-	cloneRepoCommandSequence := fmt.Sprintf("git clone %s %s; cd %s; git checkout %s", cloneUrl, destination, destination, tag)
+	cloneRepoCommandSequence := fmt.Sprintf("git clone %s %s", cloneUrl, destination)
```

Comment:
one thing to keep in mind about this approach in general. in code build we clone from codecommit vs github. Im not saying its a huge deal that we are now cloning this from github, but its something to call out.

Reply:
yep, Im working on that actually.

Reply:
Also this is not exactly right anyway... this wouldnt work for release branches?